src/views/preview/meta.jsx

```diff
@@ -1,14 +1,23 @@
 const React = require('react');
 const Helmet = require('react-helmet').default;
+const PropTypes = require('prop-types');
 
 const projectShape = require('./projectshape.jsx').projectShape;
 
 const Meta = props => {
     const {id, title, instructions, author} = props.projectInfo;
 
-    // Do not want to render any meta tags unless all the info is loaded
-    // Check only author (object) because it is ok to have empty string instructions
-    if (!author) return null;
+    if (!author) {
+        // Project info is not ready. It's either fetching state, or logged-out users creating project.
+        if (!props.userPresent) {
+            return (
+                <Helmet>
+                    <title>Scratch - Imagine, Program, Share</title>
```

Comment:
Should this be localized?

Reply:
Currently, titles are not localized. It can be, but let's make another issue for that.